test/CAS/cache_replay_multiple_files_mccas.swift

```diff
@@ -0,0 +1,42 @@
+// REQUIRES: OS=macosx
```

Comment:
Why does this require macosx?

Reply:
MCCAS doesn't work for non-darwin platforms, so we have to guard the test for linux and windows